### DISCUSSION TITLE
Continue_story paragraph by paragraph?

### DISCUSSION THREAD

**Discussion by mlinden87** [ephread/inkgd#61] [Q&A] [answered]
Is it possible to break the execution of continue_story() after each paragraph or line break and yield until a key is pressed?

Trying to populate a dialogue box with an ink story where a key press advances the ink story, paragraph by paragraph. 

e.g. 
A: Hey mate, how's it going.
-> key press
B: Not bad, how about you?
-> key press
A: [...]

**Comment by francoisdlt** [comment]
i believe that is the current default behavior (which is the one we use in our game)...

**Comment by francoisdlt** [comment, marked as answer]
only continue_story_maximally skips until next choice : 
```
## Continue the story until the next choice point or until it runs out of
## content. This is as opposed to `continue` which only evaluates one line
## of output at a time. It returns the resulting text evaluated by the ink
## engine, concatenated together.
func continue_story_maximally() -> String:
```

**Reply by mlinden87** [reply]
Yeah, that's what made it so confusing. For some reason continue_story() behaved exactly like i expected of continue_story_maximum(). 

Ended up creating a new script, copied things over and it works as intended now. Guess it's one of those days...
